src/Arcus.Testing.Logging/Extensions/LoggerSinkConfigurationExtensions.Deprecated.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using GuardNet;
+using Serilog;
+using Serilog.Configuration;
+using Xunit.Abstractions;
+
+namespace Arcus.Testing.Logging.Extensions
+{
+    /// <summary>
+    /// Extensions on the <see cref="LoggerSinkConfiguration"/> to more easily add Serilog sinks related to logging.
+    /// </summary>
+    public static class LoggerSinkConfigurationExtensions
```

Comment:
I don't see how this relates to the title of the PR ?
Or do we have this class in another namespace as well ?

Reply:
Yes, we do. The git diff makes it a bit obscure, but what changed here is that the extension in the wrong namespace is made deprecated (this file) and that a new extension is made in the right namespace ([this file](https://github.com/arcus-azure/arcus.testing/pull/82/files#diff-7b0a6f14387d0abf2b5d6e2656841628efa9556dcf260ebb2b66bfa3f47da761R7).

Reply:
I thought it was going to be something like that.   Just missed the Obsolete marker.